Update developer_homepage.md (#32670)
DataDog/documentation: content/en/internal_developer_portal/developer_homepage.md

- **Status** (Open / Draft / Merged)  
- **Assignee / Reviewer**  

If your organization hasn't configured the GitHub integration, this section displays an empty state with a prompt to enable it from the [GitHub Integration tile][1]. To read PRs from GitHub, this integration requires the following permissions: 
- Members: Read
- Metadata: Read
- Pull Requests: Read
- Contents: Read
- Statuses: Read
- Checks: Read

> If you have multiple Github orgs connected within Datadog, users will need to have the Datadog Integrations Read Permissions to toglle between orgs.

Reviewer: ```suggestion
If you have multiple GitHub orgs connected within Datadog, users must have the Datadog Integrations Read Permissions to toggle between orgs.
```
